package.json

```diff
@@ -36,10 +36,9 @@
   },
   "scripts": {
     "test": "gulp",
-    "postinstall": "yodoctor && npm run tabtab",
```

Comment:
Hmm, I see what you mean. Pretty much what does `apt-get install git` or `brew install git`, when installing the package usually links the shell completion script to the appropriate place (usually `/etc/bash_completion.d/` for bash)

Since it's a little more involved than a regular `npm install -g` and would touch system files / settings, I initially designed it as a very explicit step to avoid any actions without user confirmation.

I might get convinced, If done right, this might be a better solution.